lib/options_test.go

```diff
@@ -259,6 +345,16 @@ func TestOptions(t *testing.T) {
 			})
 		})
 	})
+	t.Run("SummaryTrendStats", func(t *testing.T) {
+		stats := []string{"myStat1", "myStat2"}
+		opts := Options{}.Apply(Options{SummaryTrendStats: stats})
+		assert.Equal(t, stats, opts.SummaryTrendStats)
+	})
+	t.Run("RunTags", func(t *testing.T) {
```

Comment:
All of the other changes besides this one are out of the scope for this pull request. I don't mind very much, but ideally they should have been in a separate PR, which I would have reviewed and merged in a few minutes. Now they have to wait for the other more complicated things here to be fixed, and they clutter the code review and confuse the commit history a bit.

Reply:
I agree, separate PR would have been better for other tests. I can remove them from this PR if necessary.

Reply:
As I said, I don't mind all that much and they're already part of the history anyway. At this point I think it's better to leave them as they are, since the alternatives are to either force push the branch or create a new pull request from a new branch (and loose the context and comments already here)